### PR TITLE
Sign Up 

### PR DESCRIPTION
**Pull request by rita-savaliya-scaletech**
## Changes

- Win and Lose popup design
- SignUp page design and API integration

### CODE REVIEW

File: src/features/auth/component/loginForm.tsx [hunk 1]
```diff
@@ -104,9 +107,41 @@ const LoginForm: React.FC = () => {
                     <ErrorMessage
                         name='username'
                         component='p'
-                        className='text--red-400 font-size--sm pl--10 error-message mt--5'
+                        className='error font-size--sm pl--10'
                     />
                 </div>
+                {signUp && (
+                    <>
```
Comment (jasmeen-goyani-scaletech) [inline src/features/auth/component/loginForm.tsx]: Use loop on this

File: src/features/dashboard/container/dashboardContainer.tsx [hunk 2]
```diff
@@ -39,11 +47,10 @@ const Dashboard = () => {
         setDiceAnimation('rolling 4s');
 
         const params = {
-            userId: '1',
+            userId: userId,
```
Comment (jasmeen-goyani-scaletech) [inline src/features/dashboard/container/dashboardContainer.tsx]: You can use only  **userId**

File: src/shared/constants/constant.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const logInInitialValues = {
+    username: '',
+    password: '',
+};
+
+const signUpInitialValues = {
```
Comment (jasmeen-goyani-scaletech) [inline src/shared/constants/constant.ts:6]: you can use destructure **logInInitialValues** in **signUpInitialValues**

**Review by jasmeen-goyani-scaletech — CHANGES_REQUESTED** [review]

File: src/features/auth/component/loginForm.tsx (hunk 1 above)
Comment (rita-savaliya-scaletech) [reply src/features/auth/component/loginForm.tsx]: Done

File: src/features/dashboard/container/dashboardContainer.tsx (hunk 2 above)
Comment (rita-savaliya-scaletech) [reply src/features/dashboard/container/dashboardContainer.tsx]: Done

File: src/shared/constants/constant.ts (hunk 3 above)
Comment (rita-savaliya-scaletech) [reply src/shared/constants/constant.ts:6]: Done

**Review by jasmeen-goyani-scaletech — APPROVED** [review]

**Pull request merged**